### PR TITLE
Crossmatch localization

### PR DESCRIPTION
**Pull request by laurall974**
with @celiune : Add crossmatch between localizations (In case IPN detects a GRB in the GW localization region…) -> create localizations by merging multiple localizations.

### CODE REVIEW

**Comment by laurall974** [comment]
@Theodlz

File: skyportal/handlers/api/gcn.py [hunk 1]
```diff
@@ -2216,3 +2220,95 @@ async def get(self, dateobs, localization_name):
...
+            name: dateobs
+            required: true
+            schema:
+              type: dateobs
+          - in: path
+            name: localization_name
+            required: true
+            schema:
+              type: localization_name
+        responses:
```
Comment (mcoughlin) [inline skyportal/handlers/api/gcn.py:2244]: I think this should be something like:

        responses:
          200:
            description: A fits file corresponding to the intersection of the input fits files.
            content:
              application/fits:
                schema:
                  type: string
                  format: binary

**Review by mcoughlin — COMMENTED** [review]
@laurall974 A few small comments. Just about ready to merge.

**Comment by laurall974** [comment]
@mcoughlin is it ok now ?

**Comment by mcoughlin** [comment]
@laurall974 The docs are failing, I think because your spacing isn't correct. 

            description: A fits file corresponding to the intersection of the input fits files.
            content:
            application/fits:
            schema:
              type: string
              format: binary

vs.

            description: A fits file corresponding to the intersection of the input fits files.
            content:
              application/fits:
              schema:
                type: string
                format: binary

or something.

**Comment by laurall974** [comment]
Hope it is ok now !!

**Review by mcoughlin — APPROVED** [review]
Really nice! LGTM!

**Pull request merged**